examples/asr/conf/conformer/conformer_ctc_bpe.yaml

```diff
@@ -188,7 +184,7 @@ trainer:
   strategy: ddp
   accumulate_grad_batches: 1
   gradient_clip_val: 0.0
-  precision: 32 # Should be set to 16 for O1 and O2 to enable the AMP.
+  precision: 32 # 16, 32, or bf16
```

Comment:
perhaps mention here and elsewhere that bf16 is preferred if it is available (ampere..)

Reply:
I added these recommendations on top along with the table.